sequencers/modeshape-sequencer-ddl/src/test/resources/ddl/create_table_check_constraint.ddl

```diff
@@ -0,0 +1,21 @@
+--  SQL Server / Oracle / MS Access, not named
+--  Note that starting with 5.7 MySQL does not support CHECK constraints 
```

Comment:
I agree. But the comment is not correct and should be fixed or removed. I suggest removing it.